backend/experiment/views.py

```diff
@@ -76,89 +77,59 @@ def default_questions(request, rules):
-    except if ExperimentSeries.dashboard = True,
-    then all random_experiments will be returned as an array (also those with finished session)
+    ''' 
+    check which `ExperimentSeriesGroup` objects are related to the `ExperimentSeries` with the given slug
+    retrieve the group with the lowest order (= current_group)
+    return the next experiment from the current_group without a finished session
+    except if ExperimentSeriesGroup.dashboard = True,
+    then all experiments of the current_group will be returned as an array (also those with finished session)
     '''
     try:
         collection = ExperimentSeries.objects.get(slug=slug)
     except:
         return Http404
     request.session[COLLECTION_KEY] = slug
     participant = get_participant(request)
-    if collection.first_experiments:
-        experiments = get_associated_experiments(collection.first_experiments)
-        upcoming_experiment = get_upcoming_experiment(experiments, participant)
-        if upcoming_experiment:
-            return JsonResponse(
-                serialize_experiment_series(
-                    collection,
-                    redirect_to=upcoming_experiment
-                )
-            )
-    if collection.random_experiments:
-        experiments = get_associated_experiments(collection.random_experiments)
-        shuffle(experiments)
-        if collection.dashboard:
-            serialized = [
-                serialize_experiment(
-                    experiment,
-                    get_finished_session_count(experiment, participant)
-                ) for experiment in experiments]
-            return JsonResponse(
-                serialize_experiment_series(
-                    collection,
-                    dashboard=serialized
-                )
-            )
-        else:
-            upcoming_experiment = get_upcoming_experiment(
-                experiments, participant)
-            if upcoming_experiment:
-                return JsonResponse(
-                    serialize_experiment_series(
-                        collection,
-                        redirect_to=upcoming_experiment
-                    )
-                )
-    if collection.last_experiments:
-        experiments = get_associated_experiments(collection.last_experiments)
-        upcoming_experiment = get_upcoming_experiment(experiments, participant)
-        if upcoming_experiment:
-            return JsonResponse(
-                serialize_experiment_series(
-                    collection,
-                    redirect_to=upcoming_experiment
-                )
-            )
-    return JsonResponse()
-
-
-def serialize_experiment_series_group(group: ExperimentSeriesGroup):
-    grouped_experiments = GroupedExperiment.objects.filter(group_id=group.id).order_by('order')
+    active_groups = ExperimentSeriesGroup.objects.filter(
+        series=collection.id, finished=False).order_by('order')
+    if not active_groups:
+        return JsonResponse({})
```

Comment:
The `finished` relates to the group, not to a single experiment. This would only ever be set to `True` without a dashboard, so this behaves as required.